pkg/controller/machinedeployment/machinedeployment_controller_test.go

```diff
@@ -205,3 +226,65 @@ func intstrPtr(i int32) *intstr.IntOrString {
 	intstr := intstr.FromInt(int(i))
 	return &intstr
 }
+
+func expectReconcile(t *testing.T, requests chan reconcile.Request, errors chan error) {
+	t.Helper()
+
+	{
+		ctx, cancel := context.WithTimeout(context.TODO(), timeout)
+		defer cancel()
+
+	LOOP0:
```

Comment:
Should loop0 and loop1 be done in parallel instead of sequentially?

Reply:
Sure, I like that idea.